app/controllers/concerns/foreman/controller/authentication.rb

```diff
@@ -40,7 +40,11 @@ def check_user_enabled
   end
 
   def authorized
-    User.current.allowed_to?(path_to_authenticate)
+    if @available_sso.is_a?(SSO::Jwt) && @available_sso.jwt_scope&.any?
```

Comment:
it may be good to rather do `@available_sso.respond_to?(:allowed_to?)` so that any other SSO method can use scoping.

Reply:
yes, because OIDC uses JWT tokens too.